src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -172,6 +173,27 @@ class BundleSpec extends ChiselFlatSpec with BundleSpecUtils with Utils {
     }
   }
 
+  "Calling litValue on Bundle containing DontCare" should "make a decent error message" in {
+    class MyBundle extends Bundle {
+      val a = UInt(8.W)
+      val b = Bool()
+      val c = UInt(4.W)
+    }
+
+    class Example extends RawModule {
+      val out = IO(Output(new MyBundle))
+      val lit = (new MyBundle).Lit(_.a -> 8.U, _.b -> true.B)
+      out := lit
```

Comment:
```suggestion
      val out = IO(Output(new UInt()))
      val lit = (new MyBundle).Lit(_.a -> 8.U, _.b -> true.B)
      out := lit.litValue.U
```
I reflexively deleted the println, but that is where the `litValue` was 🙈.